packages/lesmis-server/src/routes/UsersRoute.ts

```diff
@@ -1,64 +0,0 @@
-/*
- * Tupaia
- * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
- *
- */
-import { Request, Response, NextFunction } from 'express';
-import uniqby from 'lodash.uniqby';
-import { Route } from '@tupaia/server-boilerplate';
-import { MeditrakConnection } from '../connections';
-import { LESMIS_PERMISSION_GROUPS } from '../constants';
-
-const FIELDS: Record<string, string> = {
-  user_id: 'id',
-  'user.creation_date': 'createdAt',
-  'user.first_name': 'firstName',
-  'user.last_name': 'lastName',
-  'user.email': 'email',
-  'user.mobile_number': 'mobileNumber',
-  'user.employer': 'employer',
-  'user.position': 'position',
-  'entity.name': 'entityName',
-  'entity.code': 'entityCode',
-  'permission_group.name': 'permissionGroupName',
-  id: 'userEntityPermissionId',
-};
-
-// Remove duplicate records and convert user keys to nice camelCase keys for the front end
-const getProcessedUserData = (userData: { user_id: number }[]) =>
-  uniqby(userData, 'user_id').map((user: Record<string, string[]>) => {
-    const newUser: Record<string, string[]> = {};
-    const keys = Object.keys(user);
-    keys.forEach(key => {
-      const newKey = FIELDS[key];
-      newUser[newKey] = user[key];
-    });
-    return newUser;
-  });
-
-export class UsersRoute extends Route {
-  private readonly meditrakConnection: MeditrakConnection;
-
-  constructor(req: Request, res: Response, next: NextFunction) {
-    super(req, res, next);
-
-    this.meditrakConnection = new MeditrakConnection(req.session);
-  }
-
-  async buildResponse() {
-    const userData = await this.meditrakConnection.getUserEntityPermissions({
-      pageSize: '10000', // retrieve all entries, as we don't handle pagination
-      columns: JSON.stringify(Object.keys(FIELDS)),
-      sort: JSON.stringify(['user.creation_date DESC']),
-      filter: JSON.stringify({
-        'entity.code': 'LA',
```

Comment:
Does the new users table within LESMIS filter to only Laos users at all?

Reply:
No there isn't. That reminds me that there also isn't any filtering of any of the data such as survey responses either. I assumed that we would do that at the permissions level and I saw some refinement notes to that extent on the BE ticket but I was meaning to check.

Reply:
Yep for survey responses they'll only see those they have permission to, but that does include all responses to public surveys (e.g. Basic Clinic Data etc.) which will be a bit weird. @enunan how much effort should we put into this? Obviously once it's their own deployment in Laos it wouldn't have any of that data on it anyway (if we ever do that), but in the meantime, the stuff they see through the LESMIS admin panel will include any Tupaia data that they have permission to see